notebooks/data_processing.ipynb

```diff
@@ -0,0 +1,795 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    new_data = new_data.sel(location_id=~new_data.location_id.isin(locations_to_remove))
</pre><p>I'd suggest to use direct index instead of boolean index. You can remove these locations from a list of all locations and then directly pass the remaining locations to locationd_id.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/kdd24/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>